#! /bin/sh /usr/share/dpatch/dpatch-run
## 01-remove-strip-from-makefile.dpatch by Daniel Moerner <dmoerner@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove strip from src/Makefile as dh_strip does this for us.

@DPATCH@
diff -urNad tint2~/src/Makefile tint2/src/Makefile
--- tint2~/src/Makefile	2008-09-05 15:47:25.000000000 -0700
+++ tint2/src/Makefile	2008-09-06 07:49:12.057562687 -0700
@@ -18,7 +18,6 @@
 
 $(PROGNAME): $(FILES)
 	$(CC) $(CFLAGS) $(FLAGS) -o $(PROGNAME) $(FILES)
-	strip $(PROGNAME)
 
 install:
 	mkdir -p $(BINDIR)
